GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -302,7 +313,12 @@ public InstanceData parseInstance(Sheet sheet, int startRow,
 
                 long iteration = 1;
                 if (hasIterationColumn) {
-                    iteration = (long) iterationRow.getCell(1).getNumericCellValue();
+                    Cell cell = iterationRow.getCell(1);
+                    if (cell == null) {
+                        iteration = 1;
```

Comment:
`iteration` is already set to `1` by default, so maybe you only need the `cell != null` case here?